### ISSUE TITLE
Succinylacetone high in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11241]
**Preferred term label:**
Succinylacetone high in blood

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK1515/ says "increased succinylacetone concentration in the blood and urine"
 and https://newbornscreening.hrsa.gov/conditions/tyrosinemia-type-i makes it sound like blood is done for newborn screening.

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001410
- label: Elevated circulating succinylacetone concentration
- parents: [Abnormal circulating carboxylic acid concentration (HP:0004354)](https://hpo.jax.org/app/browse/term/HP:0004354)
- pmid: PMID:16448836|PMID:32832707
- def: The concentration of succinylacetone in the blood circulation is above the upper limit of normal.
- comment: Succinylacetone can be created by the oxidation of glycine, and is a precursor of methylglyoxal. Succinylacetone is an abnormal tyrosine metabolite that arises from defects in the enzyme called fumarylacetoacetase.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11241
- synonyms: Succinylacetone high in blood(NONE)
